template/node/index.js

```diff
@@ -23,9 +28,9 @@ getStdin().then(val => {
 });
 
 const isArray = (a) => {
-    return (!!a) && (a.constructor === Array);
```

Comment:
Curious why you made this change? cc @padiazg

Reply:
These functions are being called after my newly added falsy check so i thought that that the !!a is redundant.
But i guess its not related to the actual issue so i should not have removed them.

Reply:
Thank you